src/overmap.cpp

```diff
@@ -3170,6 +3203,11 @@ oter_id overmap::get_default_terrain( int z ) const
     return settings->default_oter[OVERMAP_DEPTH + z].id();
 }
 
+// underlying bitset default constructs to all 0.
+static std::shared_ptr<map_data_summary> impassable_omt( new map_data_summary() );
+static std::shared_ptr<map_data_summary> passable_omt( new map_data_summary(
+            ~impassable_omt->passable ) );
```

Comment:
This is what the 'nonempty 0 alloc shared_ptr trick' is best for, because below you make a ton of increfs on the statics.

```suggestion
static map_data_summary impassable_omt{};
static map_data_summary passable_omt{ ~impassable_omt->passable ) };
```